source/val/validate_instruction.cpp

```diff
@@ -483,6 +483,22 @@ spv_result_t InstructionPass(ValidationState_t& _, const Instruction* inst) {
     if (auto error = LimitCheckNumVars(_, inst->id(), storage_class)) {
       return error;
     }
+  } else if (opcode == SpvOpSamplerImageAddressingModeNV) {
```

Comment:
Should this also check whether another SpvOpSamplerImageAddressingModeNV instruction has been seen? The extension isn't explicit, but I assume only a single instance of this instruction can be specified.

Reply:
yes only one is allowed. I can add a check